cloud/logger/src/Logger.ts

```diff
@@ -0,0 +1,8 @@
+interface Logger {
+  debug(message: string, data?: any, source?: string, tags?: string[]): void;
```

Comment:
what is source, tags ? I think that source is actually determined in logger constructor.
I would like to see documentation for this to fully know what we talking about here.

Can we add generic method as well `log(level: ErrorLevel, message..`